.github/workflows/ci.yml

```
@@ -0,0 +1,154 @@
+name: "[CI] Test"
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```

Wouldn't it be easier to just do?

```suggestion
on: "push"
```